style.css

```diff
@@ -647,6 +785,136 @@
     border-radius: 16px;
   }
 
+  #works_popup_desktop {
+    display: none;
+  }
+
+  #works_popup_mobile {
+    visibility: hidden;
+    position: fixed;
+    width: 100%;
+    height: 100vh;
+    top: 0;
+    background-color: rgba(0, 0, 0, 0.4);
+    padding: 0;
+    overflow-y: scroll;
+  }
+
+  #works_popup_tonic {
+    visibility: hidden;
+    position: fixed;
+    width: 100%;
+    height: 100vh;
+    top: 0;
+    background-color: rgba(0, 0, 0, 0.4);
+    padding: 0;
+    overflow-y: scroll;
+  }
+
+  .project1_popup {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: #fff;
+    border-radius: 16px;
+    margin: 20px;
+    opacity: none;
+  }
+
+  .popup_header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    align-items: center;
+    margin-top: 0;
+  }
```

Comment:
- [ **OPTIONAL** ] **RECOMENDED** I also suggest you increase the **@media query breakpoint** for your Popup even for the whole project as it generates some styling issues on some screen sizes. I think this value of (`min-width: 768px`) that was required for your Desktop version milestone will be perfect.
Look like the image below:
![Screenshot from 2022-07-20 12-45-29](https://user-images.githubusercontent.com/83042742/179963866-d8ff0fa5-35a9-47eb-992d-4e823288c838.png)